lib/linter-rubocop.coffee

```diff
@@ -9,7 +9,8 @@ class LinterRubocop extends Linter
 
   # A string, list, tuple or callable that returns a string, list or tuple,
   # containing the command line (with arguments) used to lint.
-  cmd: 'rubocop --force-exclusion --format emacs'
+  cmd: atom.config.get("linter-rubocop.cmd") ||
+         'rubocop --force-exclusion --format emacs'
```

Comment:
I do not fully understand your question. This line is supposed to keep the default behaviour in the case someone doesn't introduce or introduce a falsy cmd in the `config.cson`.

It might be that `atom.config.get` is already figuring out the specified default value in the `init.coffee` file.